File: app/src/main/kotlin/com/ohdroid/zbmaster/application/BaseApplication.java
package com.ohdroid.zbmaster.application;

import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;
import com.ohdroid.zbmaster.BuildConfig;
import com.ohdroid.zbmaster.application.di.ApplicationComponent;
import com.ohdroid.zbmaster.application.di.ApplicationModule;
import com.ohdroid.zbmaster.application.di.DaggerApplicationComponent;
import com.ohdroid.zbmaster.homepage.areaface.model.FaceInfo;
import com.tencent.tauth.Tencent;

import cn.bmob.v3.Bmob;

/**
 * Created by ohdroid on 2016/2/28.
 */
public class BaseApplication extends Application {

    private ApplicationComponent appComponent;

    @Override
    public void onCreate() {
        super.onCreate();
        Bmob.initialize(this, BuildConfig.BMOB_APP_KEY);
        Fresco.initialize(this);
        appComponent = DaggerApplicationComponent.builder().applicationModule(new ApplicationModule(this)).build();
    }

    public ApplicationComponent getApplicationComponent() {
        return appComponent;
    }
}

File: app/src/main/kotlin/com/ohdroid/zbmaster/application/BaseApplication.kt
package com.ohdroid.zbmaster.application

import android.app.Application
import cn.bmob.v3.Bmob
import com.ohdroid.zbmaster.BuildConfig

/**
 * Created by ohdroid on 2016/2/24.
 *
 * 全局设置:
 * 1.导入Bmob后台
 */
class BaseApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Bmob.initialize(this, BuildConfig.BMOB_APP_KEY);
    }
}
